geomstats/special_orthogonal_group.py

```diff
@@ -342,23 +342,23 @@ def skew_matrix_from_vector(self, vec):
                 [[0., 1., 0.],
                  [-1., 0., 0.],
                  [0., 0., 0.]]
-                ])
+                ]]*vec.shape[0])
 
-            basis_vec_1 = gs.array([[1., 0., 0.]])
-            basis_vec_2 = gs.array([[0., 1., 0.]])
-            basis_vec_3 = gs.array([[0., 0., 1.]])
+            basis_vec_1 = gs.array([[1., 0., 0.]]*vec.shape[0])
+            basis_vec_2 = gs.array([[0., 1., 0.]]*vec.shape[0])
+            basis_vec_3 = gs.array([[0., 0., 1.]]*vec.shape[0])
             cross_prod_1 = gs.einsum(
-                'ijk,ni,nj->nk',
+                'hijk,hi,hj->hk',
```

Comment:
NIT: Can you use `n` instead of `h`: this is the index we use to denote the sample's indices, for code readability. Thanks!